definitions/infra-host/definition.yml

```diff
@@ -79,6 +79,47 @@ synthesis:
         host.image.id:
         host.image.version:
         instrumentation.provider:
+
+    # host information from log data
+    - identifier: host
```

Comment:
Are those changes part of this PR on purpose?

Reply:
no, something is wrong here

Reply:
fixed!